wgsl/index.bs

```diff
@@ -2683,7 +2683,15 @@ Issue: Which index is used when it's out of bounds?
        <td>The result is the value of the |i|'<sup>th</sup> element of the array value |e|.<br>
            The first element is at index |i|=0.<br>
            If |i| is outside the range [0,|N|-1], then an index in the range [0, |N|-1] is used instead.<br>
-           (OpCompositeExtract)
```

Comment:
would it make sense to still say it's either `OpCompositeExtract` or `OpAccessChain`?

Reply:
The expression will most always directly result in a load or store too, but that seems to go too far. If the mapping is not explained how it is determined, I'm not sure I see the value in mentioning it.